### PR TITLE
Add support of verify function of zks crypto library

### PR DESCRIPTION
**Pull request by MaximFischuk**

### CODE REVIEW

File: src/main/java/io/zksync/sdk/zkscrypto/lib/ZksCrypto.java [hunk 1]
```diff
@@ -115,4 +115,10 @@ public ZksSignature signMessage(final ZksPrivateKey privateKey, byte[] message)
         }
     }
 
+    public boolean verifyMessage(final ZksPackedPublicKey publicKey, final ZksSignature signature, final byte[] message) throws Exception {
```
Comment (ly0va) [inline src/main/java/io/zksync/sdk/zkscrypto/lib/ZksCrypto.java:118]: Let's stick to the name `verifyMusig` to distinguish it from other signature formats and verifications

Comment (MaximFischuk) [reply src/main/java/io/zksync/sdk/zkscrypto/lib/ZksCrypto.java:118]: Think `signMessage` should be renamed too, right?

Comment (ly0va) [reply src/main/java/io/zksync/sdk/zkscrypto/lib/ZksCrypto.java:118]: Sure, let's name it `signMusig` for consistency

**Comment by ly0va** [comment]
I thought we settled on `signMessage` and `verifyMessage` to avoid breaking changes :sweat_smile:

**Review by ly0va — APPROVED** [review]
Didn't want to bother you with the naming for third time :smile:

**Pull request merged**